### DISCUSSION TITLE
benchmark socket limit > 1018? ubuntu 20.04

### DISCUSSION THREAD

**Discussion by hst-m** [uNetworking/uWebSockets#1159] [Q&A] [answered]
when I run [benchmark](https://github.com/uNetworking/uWebSockets/blob/master/benchmarks/load_test.c) with connection size greater than 1018 `./load_test 2000 localhost 3001 0` [getaddrinfo](https://github.com/uNetworking/uSockets/blob/master/src/bsd.c#L258) in uSockets throws System error "Device or resource busy" when opening the 1019 connection and it stops running. Any idea how to fix this? Ubuntu 20.04

getaddrinfo source for this error:
https://github.com/bminor/glibc/blob/master/sysdeps/posix/getaddrinfo.c#L887-L897
"The root cause could .. be a resource allocation failure .. return EBUSY"

**Comment by ghost** [comment, marked as answer]
Probably ulimit related, or similar. Increase allowed FDs
